frontend/src/Pages/MyProgress.tsx

```diff
@@ -48,9 +75,20 @@ export default function MyProgress() {
                         </div>
                         <div className="flex flex-row gap-12 mt-12">
                             <div className="card bg-base-teal h-[531px] w-[60%] p-4 overflow-y-auto">
-                                <div className="flex flex-row">
-                                    <h2>All Courses</h2>
-                                    {/* dropdown will go here */}
+                                <div className="flex flex-row gap-x-4">
```

Comment:
Instead of using `gap-x-4` for the row with the dropdowns, would you be able to do `justify-between`? It just puts them on opposite sides which I think looks slightly better. Same with the certificates dropdown